interface/eRx_xml.php

```diff
@@ -784,7 +785,7 @@ function PatientMedication($doc,$r,$pid,$med_limit)
             $b->appendChild( $sig );
             $dispenseNumber = $doc->createElement( "dispenseNumber" );
             $dispenseNumber->appendChild(
-                $doc->createTextNode( $prec['quantity'] )
+                $doc->createTextNode( $prec['quantity'] )  // THIS CAN't BE RIGHT tdm $prec is outside of scope
```

Comment:
That is from the function/query above this one.  Totally out of scope, and was probably meant to be $presc vs. prec.  Very confusing for a while there.  Probably by the same person that used JASON vs. JSON.  :)  Frankly, I am lost as to what this function does, unless it is nested...  $active above starts a query string with AND.  Makes me think it is a nested function, which may mean that $prec is in scope in a sort of slimy, stupid dangerous way.